mitosheet/src/mito/components/endo/ColumnHeader.tsx

```diff
@@ -171,7 +172,8 @@ const ColumnHeader = (props: {
                 'endo-column-header-text',
                 {
                     'endo-column-header-container-selected': selected,
-                    'recon': isColumnCreated || isColumnRenamed
+                    'recon': isColumnCreated || isColumnRenamed,
+                    'mito-search-match': matchesSearch,
```

Comment:
I don't see this class anywhere. Forget to commit it?

Reply:
Oh yeah I think that's from exploring the "jump to next match" functionality - that can be removed from this PR

Reply:
Although maybe it'd be good to have a special background color or something? Not sure how prominent we want the highlighting to be